client/app/sources/EventSource.js

```diff
@@ -29,6 +29,9 @@ const EventSource = {
   },
   confirm(conferenceRoomId, eventId) {
     return axios.post(`${eventPath(conferenceRoomId)}/${eventId}/confirm`);
+  },
+  finish(conferenceRoomId, eventId) {
+    return axios.post(`${eventPath(conferenceRoomId)}/${eventId}/finish`);
```

Comment:
How about:
const finishPath = (conferenceRoomId, eventId) => `${eventPath(conferenceRoomId)}/${eventId}/finish`

and then:
return axios.post(finishPath(conferenceRoomId, eventId));